src/relayclient/RelayClient.ts

```diff
@@ -86,24 +86,40 @@ export class RelayClient {
    *
    * @param {*} transaction - actual Ethereum transaction, signed by a relay
    */
-  async _broadcastRawTx (transaction: Transaction): Promise<{ receipt?: TransactionReceipt, broadcastError?: Error, wrongNonce?: boolean }> {
+  async _broadcastRawTx (transaction: Transaction): Promise<{ hasReceipt: boolean, broadcastError?: Error, wrongNonce?: boolean }> {
```

Comment:
This method only returns for tests... Weird. Very weird.

Reply:
yes, the receipts return value is only for testing.